plugin/federation/testdata/entities/nokey.yml

```diff
@@ -4,3 +4,6 @@ exec:
   filename: testdata/entities/generated/exec.go
 federation:
   filename: testdata/entities/generated/federation.go
+
+autobind:
+  - "github.com/99designs/gqlgen/plugin/federation/test_data/model"
```

Comment:
It would be good to have a separate test with autobind.  Maybe have a different `nokey-autobind.yml` so that we can ensure these tests work with model autogen as well as autobinding.  WDYT?